org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/serializer/SerializerFragment2.xtend

```diff
@@ -461,8 +460,9 @@ class SerializerFragment2 extends AbstractStubGeneratingFragment {
 				@�Inject�
 				protected void init(�IGrammarAccess� access) {
 					�IF partitions.size > 1�
+						grammarAccess = (�grammar.grammarAccess�) access;
```

Comment:
Nit: Can be moved above the if (the else branch needs the same code)

Reply:
done